packages/account/src/Sections/Verification/ProofOfOwnership/proof-of-ownership-form.jsx

```diff
@@ -0,0 +1,244 @@
+import classNames from 'classnames';
+import React, { useRef, useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Button, useStateCallback } from '@deriv/components';
+import { Formik } from 'formik';
+import { localize } from '@deriv/translations';
+import FormFooter from '../../../Components/form-footer';
+import FormBody from '../../../Components/form-body';
+import FormSubHeader from '../../../Components/form-sub-header';
+import FormBodySection from '../../../Components/form-body-section';
+import { isMobile, readFiles, WS, DOCUMENT_TYPE } from '@deriv/shared';
+import Card from './card.jsx';
+import DocumentUploader from '@binary-com/binary-document-uploader';
+import { IDENTIFIER_TYPES, VALIDATIONS } from './constants/constants';
+
+const getScrollOffset = (items_count = 0) => {
+    if (isMobile()) return '200px';
+    if (items_count <= 2) return '0px';
+    return '80px';
+};
+const ProofOfOwnershipForm = ({
+    client_email,
+    grouped_payment_method_data,
+    refreshNotifications,
+    total_documents_required,
+    updateAccountStatus,
+}) => {
+    const grouped_payment_method_data_keys = Object.keys(grouped_payment_method_data);
+    const initial_values = {};
+    const [is_submit_button_disabled, setIsSubmitButtonDisabled] = React.useState(true);
+    const [form_state, setFormState] = useStateCallback({ should_show_form: true });
+    const form_ref = useRef();
+    const [document_upload_errors, setDocumentUploadErrors] = useState();
+    useEffect(() => {
+        setDocumentUploadErrors([]);
+    }, []);
+    const fileReadErrorMessage = filename => {
+        return localize('Unable to read file {{name}}', { name: filename });
+    };
+
+    const validateFields = values => {
+        const errors = {};
+        errors.data = [...document_upload_errors];
+        let total_documents_uploaded = 0;
+        let has_errors = false;
+        const cards = values?.data;
+        Object.keys(cards)?.forEach?.(card_key => {
+            const items = cards?.[card_key] ?? {};
+            errors.data[card_key] = errors.data?.[card_key] ?? [];
+            Object.keys(items)?.forEach?.(item_key => {
+                errors.data[card_key][item_key] = errors.data?.[card_key]?.[item_key] ?? {};
```

Comment:
and here the same question , but now we use empty object for reassigning

Reply:
@yauheni-kryzhyk-deriv, I do the same here, because I want to retain the previous history

The errors are attached to a particular control as seen in the screenshot below, and those controls are all dynamic, meaning one payment method might have `1` set of controls and another might have `10` or more. I'll need to display all those errors if any

<img width="953" alt="invalid poo state" src="https://user-images.githubusercontent.com/104334373/202370643-1b12e3ff-fa6d-422f-a048-5bd0978926d7.png">